### ISSUE TITLE
Bug generating Sitemap Cron expression

### ISSUE THREAD

**Issue by barbazul** [magento/magento2#1020]
I just found this in M1, surprised to see it is also in M2.

app/code/Magento/Cron/Model/Config/Backend/Sitemap.php line 68 reads

```php
$frequency = $this->getData('groups/generate/frequency/value');
```

And should read

```php
$frequency = $this->getData('groups/generate/fields/frequency/value');
```

This causes $frequency to be always false and so, sitemap is always generated daily regardless of configuration

Edit: Typo

**Comment by otoolec** [comment]
Thanks for submitting this bug, the internal ticket is MAGETWO-33637

**Comment by snohe** [comment]
@barbazul, this has been fixed in 0.74.0-beta1.  Thank you again for submitting this issue.  Should you have any other issues, please don't hesitate to contact us.
